.github/actions/okfn-contributors/Dockerfile

```
@@ -0,0 +1,11 @@
+FROM node:20
+LABEL mantainer="nkzren"
```

Não é uma informação que temos utilizado, por mim poderíamos remover. Mas se for pra colocar, deixaria com a manutenção da OKBR
```suggestion
LABEL maintainer="Open Knowledge Brasil"
```